src/test/scala/chisel3/experimental/tests/VerilatorBasicTests.scala

```diff
@@ -8,7 +8,7 @@ import chisel3.tester.internal.VerilatorBackendAnnotation
 import org.scalatest._
 
 class VerilatorBasicTests extends FlatSpec with ChiselScalatestTester with Matchers {
-  behavior of "Testers2"
+  behavior of "Testers2_Verilator"
```

Comment:
Maybe cleaner as "Testers2 with Verilator"? I don't think you need an underscore in quotes?